### PR TITLE
DocumentCard: Fix color and font-size issues

### PR DESCRIPTION
**Pull request by atneik**

### CODE REVIEW

**Comment by cliffkoh** [comment]
Looks pretty straightforward. Approved.

File: src/components/DocumentCard/DocumentCardTitle.scss [hunk 1]
```diff
@@ -1,9 +1,12 @@
 @import '../../common/common';
 
+$documentCard-titleSize: 16px;
```
Comment (dzearing) [inline src/components/DocumentCard/DocumentCardTitle.scss]: should "16px" font size come from a defined variable?

**Comment by dzearing** [comment]
Approved with minor suggestion.

File: src/components/DocumentCard/DocumentCardTitle.scss (hunk 1 above)
Comment (atneik) [inline src/components/DocumentCard/DocumentCardTitle.scss]: In Fabric we have 17px(size-l) or 15px(size-m-plus) defined, but no 16px. This case is an abnormally where we have to use 16px.

**Pull request merged**